### PR TITLE
Flanification

### PR DESCRIPTION
**Pull request by sjmiles**
- Revamp live state to be per-Flan instead of per-Layer, to support cross-layer bindings.
- Significant refactoring
- Bug fixes
- Switching back to non-Worker mode for Build and Run

### CODE REVIEW

File: pkg/Library/Graph/Atoms/GraphAgent.js [hunk 1]
```diff
@@ -35,40 +34,37 @@ async update(inputs, state, tools) {
...
+      publicGraphs: parsed.map(g => g.meta ? g : values(g)?.map(gg => JSON.parse(gg))).flat()
+    });
+  }
 },
-formatFetchPublishGraphsUrl(publishedGraphsUrl) {
+formatFetchUrl(publishedGraphsUrl) {
   return `${publishedGraphsUrl}/${this.publicGraphsPath}.json`;
 },
-formatPutPublishGraphUrl(graphId, owner, {publishedGraphsUrl}) {
+formatPutPUrl(graphId, owner, {publishedGraphsUrl}) {
```
Comment (mariakleiner) [inline pkg/Library/Graph/Atoms/GraphAgent.js:58]: typo - should be `formatPutUrl`

File: pkg/Library/Graphs/Demos/AIClock.js [hunk 2]
```diff
@@ -0,0 +1,364 @@
+export const graph = {
```
Comment (mariakleiner) [inline pkg/Library/Graphs/Demos/AIClock.js:1]: there is also `pkg/GraphsDemos/` folder... i think there should only be one?

File: pkg/Run/web/main.js [hunk 3]
```diff
@@ -3,53 +3,54 @@
...
+  };
+  //log(graph);
+  //
+  const {services} = await loadLibraries(graph.meta, await Persist.restoreValue('$UserSettings$settings$userSettings'));
+  flan.services = services;
+  //
+  // create layer
+  await flan.createLayer([baseGraph, graph], '');
+  // ready;
+  log('app is live 🌈');
```
Comment (mariakleiner) [inline pkg/Run/web/main.js:51]: should be `flan` is live :)

**Review by mariakleiner — APPROVED** [review]
omg, so much stuff 🥇

File: pkg/Library/Graphs/Demos/AIClock.js (hunk 2 above)
Comment (sjmiles) [reply pkg/Library/Graphs/Demos/AIClock.js:1]: Yes, but I wasn't sure if you had dependencies on the old one.

File: pkg/Run/web/main.js (hunk 3 above)
Comment (sjmiles) [reply pkg/Run/web/main.js:51]: Fixed
```Flan is ready 🍮```

File: pkg/Library/Graphs/Demos/AIClock.js (hunk 2 above)
Comment (mariakleiner) [reply pkg/Library/Graphs/Demos/AIClock.js:1]: sorry, i missed this comment. no, i don't think there are dependencies on the Graphs/Demos files in general, these are just a precaution against losing them (maybe they shouldn't be in this repo at all? dunno)...
anyway, i'll remove the duplicates in a followup PR.

**Pull request merged**